package.json

```diff
@@ -45,6 +45,7 @@
 		"deepdash": "^5.3.9",
 		"deepmerge": "^4.2.2",
 		"lodash": "^4.17.21",
+		"mout": "^1.2.4",
```

Comment:
After all the pain of having to migrate away from mout, it's probs best not to reintroduce it 😅 

Do what is done in falafel and copy the mout source code for the methods you need (you could just copy the mout utils and test file from falafel).

Reply:
But that was because of a vulnerabilities in previous versions where we couldn't update it because it was using by our dependencies right? 🤔

Reply:
yh, and I don't want a future repeat of that 🙃